### PR TITLE
feature/conduit behind CLI flag

### PR DESCRIPTION
**Pull request by kschrief**

### CODE REVIEW

File: server/src/conduit.mjs [hunk 1]
```diff
@@ -31,45 +31,92 @@ import * as commandLine from './commandLine.mjs';
...
+
+      conduitWs.clientIsAlive = false;
+      const oConduitMsg = {
+        from: 'mock-firebolt',
+        type: 'PING-FROM-SERVER',
+        data: undefined
+      };
+      const conduitMsg = JSON.stringify(oConduitMsg);
+      conduitWs.send(conduitMsg);
+      //console.log('Conduit WebSocket sent a ping message to client');
```
Comment (vyvanand) [inline server/src/conduit.mjs:67]: This can be comment rather than commenting console.log line.

File: server/src/commandLine.mjs [hunk 2]
```diff
@@ -122,5 +124,6 @@ export {
   httpPort, socketPort,
   conduitSocketPort, conduitKeySocketPort,
   developerToolPort, developerToolName,
-  enabledSdkNames, enabledTriggerPaths, proxy
+  enabledSdkNames, enabledTriggerPaths, proxy,
+  conduit
```
Comment (vyvanand) [inline server/src/commandLine.mjs:128]: readme for conduit command line passing is missing. What command we need to run to start conduit?

File: server/src/conduit.mjs (hunk 1 above)
Comment (kschrief) [reply server/src/conduit.mjs]: Addressed

**Review by vyvanand — DISMISSED** [review]

**Review by dhillomk — DISMISSED** [review]

**Review by michaelcfine — APPROVED** [review]
Just quickly glanced at this, but looks good.

**Review by vyvanand — APPROVED** [review]

**Review by dhillomk — APPROVED** [review]

**Pull request merged**